Split out Docker-based value-add stuff to separate repo (#54)
peterbourgon/fastly-exporter: README.md

docker pull mrnetops/fastly-exporter
```

> See [TODO][todo] for a complete docker-compose based out-of-the-box experience,
> including pre-built Grafana dashboards.
> 
> [todo]: https://example.com
> 

Author: ```suggestion
See https://github.com/mrnetops/fastly-exporter-docker for a Dockerfile, and
https://github.com/mrnetops/fastly-dashboards for a set of pre-built Grafana
dashboards.
```
